storage/src/store/mod.rs

```diff
@@ -0,0 +1,79 @@
+//! A simple append-only key-value store backed by a log of state-change operations.
+//!
+//! This store provides append-only functionality without requiring MMR (Merkle Mountain Range) for
+//! authentication. It's designed for efficient local storage scenarios where cryptographic proofs
+//! are not required, making it more performant than [`crate::adb::Any`] for non-authenticated use cases.
+//!
+//! # Terminology
```

Comment:
I think it is worth discussing `commit` in this section (as it supports rolling back to the last commit boundary).